tests/unit/api/test_document_builders.py

```diff
@@ -82,6 +83,25 @@ def test_build_environment_document(
     assert environment_document["webhook_config"]["secret"] == django_webhook.secret
 
 
+def test_build_environment_document__project_has_server_key_only_feature_ids__return_expected(
+    django_environment: DjangoEnvironment,
+    django_project: DjangoProject,
+    django_enabled_feature_state: DjangoFeatureState,
+) -> None:
+    # Given
+    expected_server_key_only_feature_ids = [django_enabled_feature_state.feature.id]
+    django_project.server_key_only_feature_ids = expected_server_key_only_feature_ids
```

Comment:
I guess the expectation then is that the queryset when building the environment document will have this attribute added to it via `annotate` or something?

Reply:
Yes. Do you want me to throw in an explicit test with the attribute absent?

Reply:
No, I think it's fine.